# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmBCMemory.cpp
# Commit: b9d32ef0ad83
# Full Hash: b9d32ef0ad835c014d1ced7577ed3d4c47a00031
# Author: Lars T Hansen <lhansen@mozilla.com>
# Date: 2021-10-18 16:09:29
# Regressor Bug: 1727084
# File Overlap Count: 1
# Description:
#   Bug 1727084 - Memory64 - Bulk memory operations. r=yury
#   
#   Implement all bulk memory operations on 64-bit memories.  Only
#   out-of-line paths are supported for memory.copy and memory.fill, for
#   now.
# ==============================================================================

diff -r 359e414a1d7f -r b9d32ef0ad83 js/src/wasm/WasmBCMemory.cpp
--- a/js/src/wasm/WasmBCMemory.cpp	Mon Oct 18 07:31:54 2021 +0000
+++ b/js/src/wasm/WasmBCMemory.cpp	Mon Oct 18 07:31:55 2021 +0000
@@ -1711,7 +1711,7 @@
 //
 // Bulk memory.
 
-void BaseCompiler::emitMemCopyInline() {
+void BaseCompiler::memCopyInlineM32() {
   MOZ_ASSERT(MaxInlineMemoryCopyLength != 0);
 
   int32_t signedLength;
@@ -1916,7 +1916,7 @@
   freeI32(src);
 }
 
-void BaseCompiler::emitMemFillInline() {
+void BaseCompiler::memFillInlineM32() {
   MOZ_ASSERT(MaxInlineMemoryFillLength != 0);
 
   int32_t signedLength;